content/docs/source-git/_index.md

```diff
@@ -4,6 +4,73 @@ draft: false
 weight: 1
 ---
 
+Let's talk about what source-git actually is.
+
+We initiated source-git as an alternative to working in
+[dist-git](https://github.com/release-engineering/dist-git). The need was
+especially amplified after we realized that many projects already do this. The
+problem was that everyone created a workflow and tooling for themselves which
+made it really hard to share knowledge and onboard new projects. Our intention
+is to create an implementation which is highly configurable, easy to start with
+and provdes both command-line tooling and a service which is tightly
+integration with GitHub and GitLab.
+
+Source-git is a git repository which contains upstream sources and packaging
+files for a downstream distribtions, in our case Fedora Linux, CentOS Stream
+and Red Hat Enterprise Linux.
+
+The basic requirement on a source-git repo is that one needs to be able to
+create a SRPM out of a current checkout. We need several steps to achieve such
+a thing:
+
+* RPM Spec file
+
+* Archive with sources
+
+* Spec file patches (if applicable)
```

Comment:
Under "Spec file patch" I'd imagine a patch that modifies a spec file :-) I'd call it
- patches
- downstream patches
- patches to be applied in the spec file

(you can ignore this note if you don't like any of those or don't have any better)

Reply:
I like a combination of 2) and 3) :D